core/src/replay_stage.rs

```diff
@@ -4582,7 +4588,7 @@ pub(crate) mod tests {
             &blockstore,
             &bank_forks,
             &leader_schedule_cache,
-            &rpc_subscriptions,
+            &Some(rpc_subscriptions.clone()),
```

Comment:
nit: These test changes would probably be simpler by doing `let rpc_subscriptions = Some(rpc_subscriptions);` earlier on, and then passing `&rpc_subscriptions` as before. Like at line 5193